google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -218,9 +221,11 @@ private static String getAppEngineInstanceName() {
    *  @see <a href="https://cloud.google.com/logging/docs/api/v2/resource-list">Logging Labels</a>
    */
   static class LabelLoggingEnhancer implements LoggingEnhancer {
```

Comment:
Why is this class here but not in in same package with `TraceLoggingEnhancer`?. `TraceLoggingEnhancer` is also used only in one place (this `MonitoredResourceUtil`). It looks like either both classes should be standalone public classes (like `TraceLoggingEnhancer` currently is), or both private static classes (like `LabelLoggingEnhancer` is declared).